playbooks/ansible-cloud/vcsa-appliance/files/bootstrap.yaml

```diff
@@ -1,34 +1,4 @@
 ---
 - hosts: appliance
   gather_facts: false
-  vars:
```

Comment:
we don't want to delete this, as new jobs will parent to ansible-cloud-vsca-appliance directly. So SSH key falls into still.

Reply:
I don't want to delete, I just would like to move it out of the nested Ansible. We prepare a tox env just to run a couple of tasks. This is a bit overkill.

Anyway, I reverted this part in my next iteration.

Reply:
Yes, it is overkill, but a known pattern right now with network jobs. I don't mind we refactor, we need to do all the same.

IMO, we'll get better improvement if we don't run prepare_vmware_tasks multiple times, that is cumulative cycles, for each playbook we run.